src/lib/Scenes/Artwork/Components/ArtworkExtraLinks.tsx

```diff
@@ -27,7 +27,7 @@ export class ArtworkExtraLinks extends React.Component<ArtworkExtraLinksProps> {
     const mailtoSubject = `Inquiry on ${artwork.title}`.concat(
       artwork.artist && artwork.artist.name ? ` by ${artwork.artist.name}` : ""
     )
-    Linking.openURL(`mailto:orders@artsy.net?subject=${mailtoSubject}`)
+    Linking.openURL(`mailto:specialist@artsy.net?subject=${mailtoSubject}`)
```

Comment:
We need to add some logic here--

If the artwork is available for BNMO, this should go to `orders@` but if it's in an auction it goes to `specialist@`.

Reply:
Ah I see, I thought that was maybe the case but it wasn't in the ticket :) Will update now.